### PR TITLE
Statistical Factsheets Table component and API changes

### LINKED ISSUE #349: StatisticalFactsheets: CarbonGrowingStock

### LINKED ISSUE #350: StatisticalFactsheets: PrimaryDesignatedManagementObjective

### PR DESCRIPTION
**Pull request by sorja**
Resolves #349 
Resolves #350 

Also contains API changes

### CODE REVIEW

File: webapp/app/statisticalFactsheets/components/table/table.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    return null
+  }
+
+  return (
+    <table className="fra-table">
+      <thead>
+        <tr>
+          {tableHeads.map((key) => (
+            <th key={key} className="fra-table__header-cell">
+              {i18n.t(key)}
```
Comment (minotogna) [inline webapp/app/statisticalFactsheets/components/table/table.js:24]: do we have the labels in en.js?

File: server/statisticalFactsheets/statisticalFactsheetsRepository.js [hunk 2]
```diff
@@ -10,6 +10,119 @@ const getStatisticalFactsheetTableAgg = async (schemaName) => {
...
 
+const getCarbonAndGrowingStock = async (schemaName, level) => {
+  const result = await db.query(`
+    SELECT *
+    FROM ${schemaName}.statistical_factsheets_view
+    WHERE level   = '${level}'
+    AND( row_name = 'carbon_stock_biomass_total'
+       OR row_name = 'carbon_stock_total'
+       OR row_name = 'growing_stock_total'
+       )
```
Comment (minotogna) [inline server/statisticalFactsheets/statisticalFactsheetsRepository.js:21]: you could use IN and $ to pass parameters to query

File: common/i18n/resources/en.js [hunk 3]
```diff
@@ -19,6 +19,47 @@ module.exports.translation = {
...
+    category: {
+      title: 'Select category and level',
+      WO: 'World',
+    },
+    forestArea: {
+      title: 'Forest Area (1990 - 2020)',
+    },
+    carbonAndGrowingStock: {
+      title: 'Carbon and Growing Stock (1990 - 2020)',
+      rowName: '',
```
[the remark this answers is not in the document]
Comment (sorja) [reply common/i18n/resources/en.js:32]: rowName is used in the table header:
image

Having it as empty will hide it.

[the remark this answers is not in the document]
Comment (minotogna) [reply common/i18n/resources/en.js:32]: can u just not use anything there?

File: webapp/app/statisticalFactsheets/components/table/table.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        <tr>
+          {tableHeads.map((key) => (
+            <th key={key} className="fra-table__header-cell">
+              {i18n.t(t(key))}
+            </th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {data.map((row) => (
```
[the remark this answers is not in the document]
Comment (sorja) [reply webapp/app/statisticalFactsheets/components/table/table.js:37]: will fix in next

**Pull request merged**